thinking-and-expressing-valueflows.md

```diff
@@ -11,7 +11,7 @@ If you have gone through the [Quick Start - API Explorer](quick-start.md) setup,
 
 ![a web browser with 3 panels lined up horizontally, the left has code in it, the middle is empty, and right has some developer documentation](https://i.imgur.com/14flkez.png)
 
-It is a user interface, but isn't an "end user application". It is a "graphiql" explorer, built for mostly for engaging as a developer with a ["graphql"](https://graphql.org/) application programming interface (API). However, anyone can use it, and you're encouraged you to stick with this, and see what you can do. [https://github.com/graphql/graphiql](https://github.com/graphql/graphiql). The "i" stands for "interactive".
+It is a user interface, but isn't an "end user application". It is a "[graphiql](https://github.com/graphql/graphiql)" explorer (the "i" stands for "interactive"), built for mostly for engaging as a developer with a ["graphql"](https://graphql.org/) application programming interface (API). However, anyone can use it, and you're encouraged to stick with this, and see what you can do.
```

Comment:
i wasn't exactly sure what this sentence was supposed to be saying:

> However, anyone can use it, and you're encouraged you to stick with this...

but we should at least remove the second 'you' 

(also took the liberty of moving the graphiql link to the mention above like how you include it for graphql)